source/testing/acceptance.md

```diff
@@ -96,6 +103,30 @@ next one starts.
   - Visits the given route and returns a promise that fulfills when all resulting
      async behavior is complete.
 
+You can use all asynchronous test helpers together with the [ES2017 feature async/await](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await) to write easy-to-read tests which
+deal with asynchronous behavior as follows:
+
+Mark the callback passed to the `test` function as asynchronous using the `async` keyword:
+
+```tests/acceptance/new-post-appears-first-test.js
+  test('should add new post', async function(assert) {
+
+  });
+```
+Before making an assertion, wait for the execution of each asynchronous helper to finish with the `await` keyword:
+
+```tests/acceptance/new-post-appears-first-test.js
+  test('should add new post', async function(assert) {
+    await visit('/posts/new');
+    await fillIn('input.title', 'My new post');
+    await click('button.submit');
+    assert.equal(this.element.querySelector('ul.posts li:first').textContent, 'My new post');
```

Comment:
We could also use `this.element.querySelector` here and make it say something akin to:

```js
assert.equal(this.element.querySelector('ul.posts li:first').textContent, 'My new post'); // alternate syntax to find('ul.posts li:first')
```